docs/tutorials/step-8.md

```diff
@@ -0,0 +1,172 @@
+# Step 8: Reusing the grader code
+
+This step explains how to separate the grader code, and eventually reuse it in other exercises. 
+
+During the grading, the file **test.ml** is evaluated in an environment that contains notably: 
+- **prelude.ml** and **prepare.ml** ;
+- the student code isolated in a module `Code` ;
+- **solution.ml** in a module `Solution` ;
+- the grading modules **Introspection**, **Report** and **Test_lib**.
+
+### Separating the grader code
+
+It is possible to extend this environment by declaring some other user-defined modules in an optional file **depend.txt**, located in the exercise directory.
+
+Each declaration in **depend.txt** is a single line containing the relative path of an *.ml* or *.mli* file. The order of the *.ml* declarations specifies the order in which each module is loaded in the grading environment.
+
+By default each dependency *foo.ml* is isolated in a module *Foo*, which can be constrained by the content of an optional signature file *foo.mli*. Furthermore, we can add an annotation `@included` at the beginning of a file *foo.ml* to denote that all the bindings of *foo.ml* is are evaluated in the toplevel environment (and not in a module *Foo*). 
```

Comment:
Please use shorter lines in text files (80 columns max.).

Here, who is "we"?

Reply:
```suggestion
By default each dependency *foo.ml* is isolated in a module *Foo*, which can be constrained by the content of an optional signature file *foo.mli*. Furthermore, we can add an annotation `@included` at the beginning of a file *foo.ml* to denote that all the bindings of *foo.ml* are evaluated in the toplevel environment (and not in a module *Foo*). 
```

Reply:
You should provide an example, especially to show what is exactly the syntax for annotations.

Reply:
No that's fine.